includes/class-paybutton-activator.php

```diff
@@ -19,6 +19,18 @@ public static function activate() {
         self::create_profile_page();
         // Set a flag to redirect the admin to the Paywall Settings page after activation
         update_option('paybutton_activation_redirect', true);
+        self::migrate_old_option();
+    }
+
+    private static function migrate_old_option() {
+        $old_value = get_option( 'paybutton_paywall_ecash_address', '' );
+        $new_value = get_option( 'pb_paywall_admin_wallet_address', '' );
+    
+        // If old_value is present and new_value is empty, copy old to new and remove old.
```

Comment:
Comments explaining the 'what' are usually pretty useless and this is a good example. Comments should cover the 'why' and then the code itself should be relatively easy to read.